share/site/duckduckgo/base.tx

```diff
@@ -13,42 +13,48 @@
 	<: if !$no_js { include "head_js.tx" } :>
 	<div class="site-wrapper <: if $homepage { :> site-wrapper--home<: } else { :> site-wrapper--static<: } :>  js-site-wrapper">
 	<: if !$no_wrapper { :>
-	  <: if !$no_header { include "header.tx" } :>
-	  <div id="content_wrapper" class="content-wrap">
-	  <: if !$no_content_internal { :>
-		<div id="content" class="content<: if !$no_cw { :>  cw<: } :>">
-			<div id="content_internal" class="content__internal">
-	  <: } :>
-		<: } else {:>
-		  <: if $homepage { :>
+		<: if !$no_header { :>
+			<: if $transparent_header { :>
```

Comment:
Is this being used anywhere?

Reply:
Not that I could see. Can probably remove!